starter-code/src/Components/CreditCard.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+class CreditCard extends Component {
+	state = {
+		type: this.props.type,
+		number: this.props.number,
+		expirationMonth: this.props.expirationMonth,
+		expirationYear: this.props.expirationYear,
+		bank: this.props.bank,
+		owner: this.props.owner,
+		bgColor: this.props.bgColor,
+		color: this.props.color
```

Comment:
Again, if your props object is have the correct keys and all the data that you need and you do not need to set state, just use props and make this a functional component.